src/server/api/routes/api-router.js

```diff
@@ -52,6 +53,11 @@ router.use('/modules', modulesRouter);
 router.use('/organizations', organizationsRouter);
 router.use('/profile', profileRouter);
 router.use('/self', selfRouter);
+
+router.use('/events_users', eventsUsersRouter2);
+
+router.use('/eventDetail', eventsDetailRouter);
```

Comment:
Why not put it under `/events/:eventId/details`?

Reply:
@tpetrina  i got 'Sorry can't find that!' when tried to use this

Reply:
@nuha90 Well, it depends on what exactly did you change and how the full route is constructed.